internal/v2/command/setup.go

```diff
@@ -25,6 +72,27 @@ func (r *CommandSide) StartSetup(ctx context.Context, iamID string, step domain.
 	return writeModelToIAM(iamWriteModel), nil
 }
 
+func (r *CommandSide) setup(ctx context.Context, step Step, iamAggregateProvider func(*IAMWriteModel) (*iam_repo.Aggregate, error)) error {
+	iam, err := r.iamByID(ctx, r.iamID)
+	if err != nil && !caos_errs.IsNotFound(err) {
+		return err
+	}
+	if iam.SetUpStarted != step.Step() && iam.SetUpDone+1 != step.Step() {
+
```

Comment:
Empty if? it not used remove, else add todo

Reply:
fixed